pkgs/by-name/li/libkazv/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+  boost,
+  cryptopp,
+  immer,
+  lager,
+  libcpr,
+  libhttpserver,
+  libmicrohttpd,
+  olm,
+  nlohmann_json,
+  pkg-config,
+  zug,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libkazv";
+  version = "0.7.0";
+
+  src = fetchFromGitLab {
+    domain = "lily-is.land";
+    owner = "kazv";
+    repo = "libkazv";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-bKujiuAR5otF7nc/BdVWVaEW9fSxdh2bcAgsQ5UO1Aw=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    boost
+    cryptopp
+    immer
+    lager
+    libcpr
+    libhttpserver
+    libmicrohttpd
+    olm
+    nlohmann_json
+    zug
+  ];
+
+  cmakeFlags = [
+    # Catch2::Catch2WithMain target isn't found
```

Comment:
Maybe because we have no catch2 in the dependencies?

Reply:
I don't use nix but https://github.com/XYenon/nur-packages has a packaging of kazv & libkazv, you might want to refer to that.

Reply:
https://github.com/XYenon/nur-packages/blob/master/pkgs/libkazv/default.nix

it's `catch2_3`, not `catch2`.

Reply:
Thanks! fixed.